### ISSUE TITLE
Map shortcode label color settings

### ISSUE THREAD

**Issue by mmwoods2019** [NEU-Libraries/drs-toolkit-wp-plugin#181]
On page "dpla map - all settings selected," I created a map shortcode. I changed the "labels" color to red, but this change is not apparent on the map. 

<img width="921" alt="screen shot 2019-02-12 at 8 11 50 am" src="https://user-images.githubusercontent.com/43454418/52637678-e9df9d00-2e9d-11e9-8116-20c168d3c68f.png">

I wonder if this is due to a global settings issue or a specific issue for the map shortcode's settings. 

I used Google Chrome Version 69.0.3497.100

**Comment by patrickmj** [comment]
Hmmmm....I haven't been able to reproduce this. From the looks of the shortcode, is it possible that you created the color labels, but didn't go back to the "Selected Items" tab to apply them? (I don't _think_ there's a way to apply them all at once?)

**Comment by mmwoods2019** [comment]
You're right! I didn't go back into selected items. But I tried it again on a different page, but the labels still all appear to be blue for me. I did this on the "Megan Test – WP Core General Template: text and images side by side" page.

<img width="794" alt="screen shot 2019-02-20 at 1 29 56 pm" src="https://user-images.githubusercontent.com/43454418/53115272-c9ee5000-3513-11e9-84d2-8ad2a0c8e55f.png">

<img width="1051" alt="screen shot 2019-02-20 at 1 29 32 pm" src="https://user-images.githubusercontent.com/43454418/53115289-d70b3f00-3513-11e9-942e-123083cab496.png">
